lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -39,7 +39,8 @@ const templateTypes = {
     selector: 'adf-card-view-textitem',
     templateUrl: './card-view-textitem.component.html',
     styleUrls: ['./card-view-textitem.component.scss'],
-    encapsulation: ViewEncapsulation.None
+    encapsulation: ViewEncapsulation.None,
+    host: { class: 'adf-card-view-textitem' }
```

Comment:
should not be text-item instead of textitem? We can also rename component's selector and files names

Reply:
closing based on https://github.com/Alfresco/alfresco-ng2-components/pull/8995#discussion_r1379735582 comment